app/src/fdroid/java/org/fossasia/susi/ai/chat/YoutubeVid.kt

```diff
@@ -0,0 +1,15 @@
+package org.fossasia.susi.ai.chat
+
+import android.content.Intent
+import android.net.Uri
+
+class YoutubeVid(chatActivity: ChatActivity) : IYoutubeVid {
+
+    var context = chatActivity
```

Comment:
Why var and why not private and why ChatActivity in constructor? You just need context